modules/tensor_mechanics/src/auxkernels/StressRecovery.C

```diff
@@ -0,0 +1,35 @@
+/****************************************************************/
```

Comment:
Some might disagree with me, but I don't really like the idea of having two separate classes for getting the stress (one for element and one for nodal values).  Could we make it so that NodalPatchRecovery can also run on element AuxVariables, and then make RankTwoScalarAux derive from it, and have it work in both cases?